.eslintrc.json

```diff
@@ -33,15 +31,14 @@
     "no-void": "error",
     "no-warning-comments": "error",
     "radix": "error",
-    "strict": "error",
-    "yoda": "error",
-    "import/imports-first": "error",
-    "import/newline-after-import": "error",
-    "import/no-duplicates": "error",
-    "import/no-mutable-exports": "error",
-    "import/no-named-as-default": "error",
-    "import/no-named-as-default-member": "error",
-    "import/order": "error",
-    "import/prefer-default-export": "error"
-  }
+    "yoda": "error"
+  },
+  "overrides": [
```

Comment:
There's some helpers file (to build the site, fetch data from MDN) that are real ESM, so we need to handle them differently.